x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -202,21 +202,21 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
         inputs: {
           type: 'nested',
           properties: {
-            type: { type: 'keyword' },
-            enabled: { type: 'boolean' },
-            processors: { type: 'keyword' },
-            config: { type: 'flattened' },
-            vars: { type: 'flattened' },
+            type: { type: 'keyword', index: false },
+            enabled: { type: 'boolean', index: false },
+            processors: { type: 'keyword', index: false },
+            config: { type: 'flattened', index: false },
+            vars: { type: 'flattened', index: false },
             streams: {
               type: 'nested',
```

Comment:
nested rings some alarm bells on my end. Especially as we have it twice. Is this nested docs in ES. If we don't need to query any parts inside inputs, lets make sure all of it is just an object with `enabled: false`. Otherwise ES will create lots of documents and slow down things.

Reply:
Just uptated the inputs to be `enabled: false`